### ISSUE TITLE
Question:

### ISSUE THREAD

**Issue by Tw1zT3d2four7** [dtankdempse/daddylive-m3u#19]
Are these m3u8 files static with no need to be updated unless a channel has had some changes or a new channel added by the provider? If so then the only thing that's a recurring process is the getEpgs.py is being ran every so often to get new guide data. Am I correct in thinking this? I am asking because i would like to set this up in house to run without using the links provided from bit.ly opting to use the actual files vs urls. is this possible? thanks to the creator of this and all who help out with it.

**Comment by dtankdempse** [comment]
Yes, the playlists are static, and I only update them as needed, such as when headers need adjustment or other changes are required.

**Comment by Tw1zT3d2four7** [comment]
> Yes, the playlists are static, and I only update them as needed, such as when headers need adjustment or other changes are required.

hey don't know if may wanna add this to you git for the standard files that need m3uproxy, but will drop it for if you wish to.

if you want to use the standard version but don't want to use a m3u proxy. you can use this ffmpeg profile in streammaster or threadfin to watch them.

-hide_banner -loglevel error -user_agent "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/129.0.0.0 Safari/537.36" -referer "https://ilovetoplay.xyz/" -headers "Origin: https://ilovetoplay.xyz/" -abort_on empty_output -re -thread_queue_size 1024 -rtbufsize 0.5MB -fix_sub_duration -analyzeduration 3500000 -probesize 4000000 -ignore_unknown -i {streamUrl} -ignore_unknown -fflags -reconnect 1 -map 0:v -map 0:a? -map 0:s? +genpts+discardcorrupt -preset ultrafast -tune zerolatency -timeout 6000000 -muxdelay 0.001 -max_interleave_delta 0 -report -copyts -c copy -report -f mpegts pipe:1
